packages/ui-react/src/containers/AccountModal/forms/CancelSubscription.tsx

```diff
@@ -10,24 +10,28 @@ import { modalURLFromLocation } from '@jwp/ott-ui-react/src/utils/location';
 import CancelSubscriptionForm from '../../../components/CancelSubscriptionForm/CancelSubscriptionForm';
 import LoadingOverlay from '../../../components/LoadingOverlay/LoadingOverlay';
 import SubscriptionCancelled from '../../../components/SubscriptionCancelled/SubscriptionCancelled';
+import { useAriaAnnouncer } from '../../AnnouncementProvider/AnnoucementProvider';
 
 const CancelSubscription = () => {
   const accountController = getModule(AccountController);
 
   const { t, i18n } = useTranslation('account');
+  const announcer = useAriaAnnouncer();
   const navigate = useNavigate();
   const location = useLocation();
   const subscription = useAccountStore((s) => s.subscription);
   const [cancelled, setCancelled] = useState(false);
   const [submitting, setSubmitting] = useState(false);
   const [error, setError] = useState<string | null>(null);
+  const expirationDate = subscription?.expiresAt ? new Date(subscription.expiresAt * 1000) : null;
```

Comment:
This creates a new Date opject on every render. I suggest moving this to the try block. Also, we use `date-fns` for date calculations/formatting, which a bit more browser-safe.

Reply:
Good to know, i'll keep this in mind for next implementations regarding date times.